allennlp/modules/transformer/transformer_module.py

```diff
@@ -32,23 +32,26 @@ def __init__(self, *args, **kwargs):
     def _get_mapping(
         cls,
         pretrained_module: Optional[torch.nn.Module] = None,
-        source="huggingface",
+        source: str = "huggingface",
         mapping: Optional[Dict[str, str]] = None,
     ):
         """
         Returns the mapping to be used, based on the optional `pretrained_module`.
         If `pretrained_module` is not given, the default module-level mapping is returned.
         """
         combined_mapping = {}
-        if "huggingface" in source:
+        if "huggingface" == source:
```

Comment:
@AkshitaB, do you sign off on this change? Is it safe? Was there a good reason you were doing a substring lookup before?

Reply:
@AkshitaB, ping?

Reply:
Yes, this is fine. Initially, the thought was that the source could be different huggingface configs, with some minor differences that could be addressed. But we haven't used it like that anywhere, so this should work fine.